src/components/ConfirmContent.tsx

```diff
@@ -139,17 +182,25 @@ function ConfirmContent({
 
                 {shouldStackButtons ? (
                     <>
+                        {shouldShowCancelButton && shouldReverseStackedButtons && (
+                            <Button
+                                style={[styles.mt4, styles.noSelect]}
+                                onPress={onCancel}
+                                large
+                                text={cancelText || translate('common.no')}
+                            />
+                        )}
```

Comment:
The `shouldReverseStackedButtons` logic seems a bit complex. We could check with design that the button have to be ordered that way

Reply:
That'd be good. @Expensify/design Can you please weigh in on if the buttons need to be ordered in the way they are currently?

<img width="357" alt="Screenshot 2024-04-09 at 2 11 42 PM" src="https://github.com/Expensify/App/assets/30054992/fd02057a-99a6-4782-b719-5fefc141560f">

Reply:
I think we want the green/success CTA at the bottom, based on @dannymcclain's mockups. From a code perspective I can't offer much - what exactly is the issue there?

Reply:
It's not a huge problem, but the code is complicated by three separate rendering conditions and it's more likely that future modifications to the code will lead to regressions. 

Though it's not a huge deal, so let's stick with the original design.

The better solution would be to refactor the buttons to a subcomponent, but there's no need to do that as part of this PR 👍

Reply:
Gotcha. Yeah we ideally want the green `Continue` button to be the easiest button to reach/click, which is why it's on the bottom.